modules/nw-ingress-aws-static-eip-nlb-configuration.adoc

```diff
@@ -0,0 +1,75 @@
+// Modules included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-aws.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-aws-static-eip-nlb-configuration_{context}"]
+= Configuring AWS Elastic IP (EIP) addresses for a Network Load Balancer (NLB)
+
+You can specify static IPs, otherwise known as elastic IPs, for your network load balancer (NLB) in the Ingress Controller. This is useful in situations where you want to configure appropriate firewall rules for your cluster network.
+
+.Prerequisites
+* You must have an installed AWS cluster.
+* You must know the names or IDs of the subnets to which you intend to map your `IngressController`.
+
+.Procedure
+
+. Create a custom resource (CR) file.
++
+Create a YAML file (e.g., `sample-ingress.yaml`) with the following content:
```

Comment:
Could these be combined?

"Create a YAML file that contains the following content:"

Or something.

Reply:
sure 🙂